src/HomePage/HomePage.js

```diff
@@ -6,18 +9,43 @@ import Container4 from './Container_4'
 import Container5 from './Container_5'
 import Container6 from './Container_6'
 
+import { HomePagePics, HomePageCaptions }  from '../pictures/HomePagePics'
 import './HomePage.css'
 
 export default function HomePage() {
+
+//pictures imported
+const pictures = HomePagePics;
+
+//display modal
+const [modalDisplay, changeModalOpen] = useState(false)
+
+//changing currently displayed polaroid on page and in modal
+const [display,toDisplay] = useState(0)
+const changeDisplay = (number) => {
+    if(number > 0) {
+        if(display === pictures.length - 1) toDisplay(0)
```

Comment:
```suggestion
      const isAtLastPicture = display === pictures.length - 1
       isAtLastPicture ? toDisplay(0) : toDisplay(display + 1)
```


Same with the below. Again, I don't know what `display` is. What is a better name for it?